app/controllers/idv/sessions_controller.rb

```diff
@@ -42,6 +42,7 @@ def init_questions_and_profile(resolution)
     end
 
     # rubocop:disable MethodLength
+    # This method is single statement spread of many lines for readability
```

Comment:
Perhaps reword this to `This method is a single statement spread across many lines for readability`?